gradle/publish.gradle

```diff
@@ -70,10 +70,10 @@ publishGhPages.dependsOn prepareGhCredentials
 
 task prepareGradlePluginsRepoRelease {
     doLast {
-        String key = buildProperties.env['GRADLE_PLUGINS_REPO_KEY'].or(buildProperties.secrets['gradle.publish.key']).string
-        String secret = buildProperties.env['GRADLE_PLUGINS_REPO_SECRET'].or(buildProperties.secrets['gradle.publish.secret']).string
-        project.setProperty('gradle.publish.key', key)
```

Comment:
I assumed this was working as @tobiasheine did the last release to Gradle Plugins Portal; does it mean that it wasn't the case?

Reply:
In that release properties were read from a file. This change was not tested.

Reply:
@mr-archano the last release was not yet automated. This is the first one.